cypress/integration/loggedOutFlows/runtimeBannerDeepLinking.spec.js

```diff
@@ -0,0 +1,76 @@
+describe('Runtime Banner Deep Linking', () => {
+  const runtimeStub = {
+    onBeforeLoad: (win) => {
+      Object.defineProperty(win.navigator, 'userAgent', {
```

Comment:
`Object.defineProperty` in the wild!!? Well there's something you don't see everyday... 👀